### PR TITLE
Solve the problem that RabbitmqMQ is disconnected when there is no message communication for a long time under permanent residence

### PR DESCRIPTION
**Pull request by flc1125**
Solve the problem that RabbitmqMQ is disconnected when there is no message communication for a long time under permanent residence.

### CODE REVIEW

**Comment by Kladislav** [comment]
same problem in long running tasks, can you merge this?

File: src/Queue/RabbitMQQueue.php [hunk 1]
```diff
@@ -30,54 +32,54 @@ class RabbitMQQueue extends Queue implements QueueContract
...
-    protected $boundQueues = [];
+    protected array $boundQueues = [];
 
     /**
      * Current job being processed.
      *
      * @var RabbitMQJob
      */
-    protected $currentJob;
+    protected RabbitMQJob $currentJob;
```
Comment (razumnyak) [inline src/Queue/RabbitMQQueue.php:77]: protected ?RabbitMQJob $currentJob;

Comment (flc1125) [reply src/Queue/RabbitMQQueue.php]: [13b29b3](https://github.com/vyuldashev/laravel-queue-rabbitmq/pull/482/commits/13b29b324c0b616aa671c76c992196afdf29dfa2)

**Comment by M-Porter** [comment]
php syntax changes should be separate from the underlying issue of this PR. If you'd like both, open two PRs.

**Pull request closed without merging**